torch/_inductor/cpp_builder.py

```diff
@@ -1449,7 +1449,7 @@ def __init__(
 
         for inc_dir in BuildOption.get_include_dirs():
             if _IS_WINDOWS:
-                self._include_dirs_args += f"/I {inc_dir} "
+                self._include_dirs_args += f'/I "{inc_dir}" '
```

Comment:
Use mslex.quote instead since it's windows. It will have better injection protection / safety